examples/scripts/chat.py

```diff
@@ -335,10 +335,13 @@ def chat_cli():
 
             chat.append({"role": "user", "content": user_input})
 
+            inputs = tokenizer.apply_chat_template(chat, return_tensors="pt", add_generation_prompt=True).to(
```

Comment:
Not too important, but I think you can pass `return_dict=True` here to get the attention mask for free.